app/script/controllers/login.controller.js

```diff
@@ -1,12 +1,13 @@
 (function () {
     angular.module('app')
-        .controller('loginController', function ($http, apiService, $scope, $location) {
+        .controller('loginController', function (apiService, $scope, $location) {
             $scope.login = function () {
+                token = localStorage;
                 apiService.login($scope.user.email, $scope.user.password)
                     .then(function (response) {
                         console.log(response);
                         apiService.getProducts();
-                        $location.path( "/" );
+                        $location.path( "/products/" );
```

Comment:
I think after login we should redirect the user to the home page, like it was before